src/genericComponents/Button.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Button } from 'react-bootstrap';
+import styled from 'styled-components';
+// import { colors, fonts } from '../theme/styleVars';
+
+const ButtonPrimary = (props: any) => {
+  const { onClick, text, variant } = props;
+
+  return (
+    <CAGButton
+      className={`${variant}-class`}
+      onClick={onClick}
+      variant={variant}
+    >
+      {text}
+    </CAGButton>
+  );
+};
+
+const CAGButton = styled(Button)`
+  {
+    border-radius: 25px;
+    box-shadow: 2px 2px 20px rgba(0, 0, 0, 29);
```

Comment:
Do you mean 0.29 for your alpha channel? The alpha channel number must be between 0 and 1